Userland/Libraries/LibWeb/MimeSniff/Resource.cpp

```diff
@@ -7,6 +7,256 @@
 #include <LibWeb/Fetch/Infrastructure/URL.h>
 #include <LibWeb/MimeSniff/Resource.h>
 
+namespace {
+
+using namespace Web::MimeSniff;
+using Byte = u8;
```

Comment:
Why use this uncommon typedef for u8? simply `u8` would be a lot more consistent with the rest of the codebase.